bindings/go-apiv2/.gitignore

```diff
@@ -0,0 +1,3 @@
+#SPDX-License-Identifier: Apache-2.0
```

Comment:
See comment on bindings/go-apiv1/.gitignore

Reply:
done in the same way as all fabric repositories....